schemas/vsds.json

```diff
@@ -150,42 +150,50 @@
                 "target_server_type": "vcenter",
                 "propertyOrder": 150
             },
+            "ipv6_script": {
```

Comment:
You will need to move the "sectionEnd" from the previous field to here for it to be in a section.

Reply:
done